src/query/storages/util/src/file_meta_data_reader.rs

```diff
@@ -40,6 +41,7 @@ impl FileMetaDataReader {
 impl Loader<FileMetaData> for Arc<dyn TableContext> {
     async fn load(
         &self,
+        _op: Operator,
```

Comment:
This op is not used?

Reply:
Yup, just for satisfying the trait fn.

Reply:
Seems weird. Maybe we can always use the input op and let the caller decide which op to use?

Reply:
fixed:)